### ISSUE TITLE
[jsdos 7.xx] can't get all message in stdout

### ISSUE THREAD

**Issue by xsro** [js-dos/emulators#1]
the message received by `events().onStdout`
<img width="520" alt="截屏2021-07-12 下午5 56 22" src="https://user-images.githubusercontent.com/64292434/125268727-90b01800-e33a-11eb-828f-906af02b47e0.png">

the dosbox canvas 
<img width="836" alt="截屏2021-07-12 下午5 58 06" src="https://user-images.githubusercontent.com/64292434/125268883-ba693f00-e33a-11eb-802a-3d7e78dfca61.png">

**Comment by caiiiycuk** [comment]
Sorry, don't get it, which message are missing?

**Comment by xsro** [comment]
for example: the full stdout is 

```
Z:\>mount c  ./
Drive C is mounted as local directory ./
```

but, We only get

```
Zmount c 
Drive C is mounted as local directory 
```

**Comment by caiiiycuk** [comment]
Well it happens here
https://github.com/js-dos/emulators/blob/c06afe5682fcec1c409cf3725520970aba6310fb/native/dosbox/src/dos/dos_files.cpp#L440

I did strip everything except alphanumeric and digits because I don’t know how to implement it correctly. Do you think adding . > \ can work for you?

**Comment by xsro** [comment]
I think things will be much better if all ASCII characters can be supported.

**Comment by caiiiycuk** [comment]
Should be fixed in emulators@0.64.0
